ccd-definition/ComplexTypes/DQ/FixedRecoverableCostsIntermediate.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "ID": "FixedRecoverableCostsIntermediate",
+    "ListElementCode": "explanationLabel",
+    "ElementLabel": "You should discuss the complexity band allocation with the other party. Read <a href='https://www.justice.gov.uk/courts/procedure-rules/civil/rules/part26' rel=\"noreferrer noopener\" target=\"_blank\">CPR26.16 Table</a></div><br /> for information on complexity bands.",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "ID": "FixedRecoverableCostsIntermediate",
+    "ListElementCode": "isSubjectToFixedRecoverableCostRegime",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Is this claim subject to the Fixed Recoverable Cost Regime requiring the court to determine a complexity band?",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "ID": "FixedRecoverableCostsIntermediate",
+    "ListElementCode": "agreedWithOtherParty",
```

Comment:
don't think a new name is needed for this, we already have a field that is similar could we change it to `complexityBandingAgreed`

Reply:
Yeah remnant from when we were gonna use a bold label,  agreedWithOtherPartywould have been the actual yes/no with empty label and an additional agreedWithOtherPartyLabel for the actual bold label. No longer need bold text, so can just reuse that  complexityBandingAgreed now